assets/base.css

```diff
@@ -1020,18 +1020,6 @@ summary::-webkit-details-marker {
   }
 }
```

Comment:
If the merchant selects `6` for the value but only has 3, should we let this happen? My instinct is yes, because thats what the setting says, but wanted to get your thoughts (or should we ensure it fills the width)

cc @wiktoriaswiecicka @nicklepine 

https://screenshot.click/25-16-788jk-7eezr.png

Reply:
Hmm I think I disagree. I think this setting should seen more as the Maximum Number of Items per row.

If there are less items in the section than the max number of items per row, they should fill the available space.

Reply:
Just noting that it can be a bit odd as well. As that would mean that we have the "magic" effect on the first row in case the amount of items is less than the limit set. But on the contrary if you have more than 6 items in this example but less than 12, the items on the second row will not take up more space but leave empty room 🤔

Reply:
@ludoboludo @nicklepine I see both of your points.  If we do go that route it means that the magic only happens if it's  the first row 🤔, which may be confusing. And also in that case wouldnt it make sense for the merchant to update the setting `Products to show row` to have a different number so it fills the space and matches the number of products?

This way we dont have any magic happening and it would look good for the merchant and we would see what the settings reflect directly.

Let me know your thoughts. Otherwise I wonder if well need a different label/language to ensure it's not confusing

Reply:
Hmm. I don't know if this is considered magic if the setting is labelled **"Maximum number of items per row"**. I don't think this setting should be prescribing the absolute number of columns in the grid on desktop, moreso the maximum number of items that can fit in a row. 

When a section contains less items than the "Maximum number of items per row", fill the space. 
When a section contains equal or more than the "Maximum number of items per row", use the prescribed maximum.

**Video explanation**:  https://screenshot.click/26-01-9qcqd-yjxkl.mp4

Reply:
@nicklepine I see what you mean 🤔 The only thing is that we are only doing this for the first row. Do you think we need to communicate that? We are only filling it for the first row and we don't for the remaining of the rows (it doesnt fill the space if needed). I see what you mean that merchants wouldnt want that, but wouldnt that be something they can change in the settings if there is less columns/products so we ensure it fills the full roll?